addons/sourcemod/scripting/ConsoleChatManager.sp

```diff
@@ -96,6 +100,7 @@ public void OnPluginStart()
 	g_cHudSymbols = CreateConVar("sm_consolechatmanager_hud_symbols", "1", "Determines whether >> and << are wrapped around the text.");
 	g_cHudType = CreateConVar("sm_consolechatmanager_hud_type", "1.0", "Specify the type of Hud Msg [1 = SendTextHud, 2 = CS:GO Warmup Timer]", _, true, 1.0, true, 2.0);
 	g_cHudHtmlColor = CreateConVar("sm_consolecharmanager_hud_htmlcolor", "#6CFF00", "Html color for second type of Hud Message");
```

Comment:
typo (chat instead of char):
- sm_consolecharmanager_hud_htmlcolor
- sm_consolecharmanager_hud_htmlcolor